collection-scripts/gather_hco

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+NS=openshift-cnv
+
+BASE_COLLECTION_PATH="/must-gather"
+
+{
+  PS4='---\n$ '
+  set -x
+  mkdir -p $BASE_COLLECTION_PATH
+  oc get pods --namespace $NS
```

Comment:
This could be also achieved by adding the namespace [here](https://github.com/kubevirt/must-gather/blob/master/collection-scripts/gather#L10):
`resources+=(ns/openshift-cnv)`
as a bonus we will get many other items like Events, ImageStreams, ... for the NS

Reply:
I'm just executing the steps the HCO team needs.
I also think that thes einformations are captured in other areas already.

Reply:
Are we ok with collecting duplicated data?

Reply:
I don't think this section adds anything on top of what is already being collected, for instance this is the output of gather-hco.txt:
```
NAMESPACE                              NAME                  NAME                  TYPE       PUBLISHER   AGE
openshift-marketplace                  certified-operators   Certified Operators   grpc       Red Hat     11d
openshift-marketplace                  community-operators   Community Operators   grpc       Red Hat     11d
openshift-marketplace                  redhat-operators      Red Hat Operators     grpc       Red Hat     11d
openshift-operator-lifecycle-manager   olm-operators         OLM Operators         internal   Red Hat     11d
NAME            CATALOG         AGE
packageserver   OLM Operators   11d
apiVersion: v1
items: []
kind: List
metadata:
  resourceVersion: ""
  selfLink: ""
NAME      VERSION                   AVAILABLE   PROGRESSING   SINCE   STATUS
version   0.0.1-2019-08-04-140707   True        False         11d     Cluster version is 0.0.1-2019-08-04-140707
NAMESPACE                              NAME                   DISPLAY          VERSION   REPLACES   PHASE
openshift-operator-lifecycle-manager   packageserver.v0.9.0   Package Server   0.9.0                Succeeded
```
The information is already gathered:
```
tree namespaces/openshift-marketplace/pods/
namespaces/openshift-marketplace/pods/
├── certified-operators-66774d7bb9-mbcvb
│   ├── certified-operators
│   │   └── certified-operators
│   │       ├── healthz
│   │       └── logs
│   │           ├── current.log
│   │           └── previous.log
│   └── certified-operators-66774d7bb9-mbcvb.yaml
├── community-operators-7dc68b99cc-qrcth
│   ├── community-operators
│   │   └── community-operators
│   │       ├── healthz
│   │       └── logs
│   │           ├── current.log
│   │           └── previous.log
│   └── community-operators-7dc68b99cc-qrcth.yaml
├── marketplace-operator-cf8bffcfd-8kdsp
│   ├── marketplace-operator
│   │   └── marketplace-operator
│   │       ├── healthz
│   │       └── logs
│   │           ├── current.log
│   │           └── previous.log
│   └── marketplace-operator-cf8bffcfd-8kdsp.yaml
└── redhat-operators-5d694f79d7-bjlgm
    ├── redhat-operators
    │   └── redhat-operators
    │       ├── healthz
    │       └── logs
    │           ├── current.log
    │           └── previous.log
    └── redhat-operators-5d694f79d7-bjlgm.yaml
```
also as @mmirecki mentioned we should use `resources+=(catalogsource)` to collect data in a standardize format instead of storing all of the output into a single file